lib/backend.js

```diff
@@ -197,7 +197,7 @@ Backend.prototype.trigger = function(action, agent, request, callback) {
     if (err) return callback(err);
     var fn = fns.shift();
     if (!fn) return callback();
-    fn(request, next);
+    fn(request, callback);
```

Comment:
Can we move this change to a separate PR — I haven't got the headspace to think about it right now (hopefully a bit more time next week), but I think:

- it's not conceptually part of this change
- it might need further discussion, and I don't want the two changes to hold one another up

Reply:
I am at conferences this and next week, so will not have time to refactor:( Will go through this right after.

Reply:
Hi @alecgibson , I am a bit puzzled as I am not sure how to create 2 separate PRs from a single fork. Any ideas?

Reply:
You should be able to just create a second branch on your fork, and raise a PR from there. You don't need to raise PRs from `master`.

Reply:
@alecgibson , thank you. Moved into a separate PR [#517].